docs/notebooks/topic_coherence-movies.ipynb

```diff
@@ -114,7 +115,7 @@
     "    # as well as pages about a single year.\n",
     "    # As a result, this preprocessing differs from the paper.\n",
     "    \n",
-    "    with open(os.path.join(data_dir, fname)) as f:\n",
+    "    with smart_open(os.path.join(data_dir, fname), 'rb') as f:\n",
```

Comment:
This looks like a bug. If you open the file in binary mode, operations like `split` and `lower` have a different meaning, compared to text.

I see the same problem in many places in this PR.